speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backendUnitTests/MediatorTests.cpp

```diff
@@ -3,21 +3,28 @@
 #include "ConfigEnums.h"
 #include "Mediator.h"
 #include "Mediator.inl"
+#include "SDAConfig.h"
 #include "mocks/DecisionMakerMock.h"
 #include <sys/types.h>
 #include <sys/stat.h>
 #define _SILENCE_EXPERIMENTAL_FILESYSTEM_DEPRECATION_WARNING 1
 #include <experimental/filesystem>
 #include "../rppUtils/RppUtils.hpp"
 
-#define MockMediator Mediator<DecisionMakerMock>
+#define MockMediator Mediator<SDecisionMakerMock>
+
+#define MockMediator2 Mediator<DecisionMakerMock<SDAConfig>>
```

Comment:
Add comments to explain what the difference between MockMediator and MockMediator2 is